service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ChildSubmissionControllerTest.java

```diff
@@ -68,18 +71,117 @@ void shouldReturnNoErrorsWhenAllDatesOfBirthAreTodayOrInPast() throws Exception
         ZonedDateTime yesterday = today.minusDays(1);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(
-            new Children(createChild(today), createChild(yesterday))
+            new OldChildren(createChild(today), createChild(yesterday))
         );
         assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
     }
 
-    private Child createChild(ZonedDateTime dateOfBirth) {
-        return new Child(null, Date.from(dateOfBirth.toInstant()), null, null, null,
+    private OldChild createChild(ZonedDateTime dateOfBirth) {
+        return new OldChild(null, Date.from(dateOfBirth.toInstant()), null, null, null,
             null, null, null, null, null, null,
-            null, null, null, null, null);
+            null, null, null, null, null, null,
+            null, null, null, null, null, null,
+            null);
```

Comment:
do you need this many lines of null? Spacing might be right, just looks off, so please check

Reply:
Lets use the builder property of the class itself